schemas/felles/customBlock.tsx

```diff
@@ -11,6 +11,7 @@ const customBlock = {
   of: [
     {
       type: SanityTyper.BLOCK,
+      styles: [{ title: 'Normal', value: 'normal' }],
```

Comment:
Her kan man sende inn en tom liste fordi normal er reservert og alltid inkludert. Kan lese mer om det [HER](https://www.sanity.io/docs/block-type#styles-23ae850fa800) 
```suggestion
      styles: [],
```

Reply:
<img src="https://media3.giphy.com/media/3o7abKhOpu0NwenH3O/giphy.gif"/>